project/Dependencies.scala

```diff
@@ -41,7 +41,7 @@ object Dependencies {
   object Compile {
     // Compile
 
-    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.13.4" exclude("org.slf4j", "slf4j-api") // ApacheV2
+    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.19.0" exclude("org.slf4j", "slf4j-api") // ApacheV2
```

Comment:
When I previously tried this update some camel test failed. I think we should do the minimum update (also to avoid risk of breaking binary compatibility). That is `2.15.6`